### PR TITLE
Add beginner exercise for Strings

### PR DESCRIPTION
**Pull request by doomspork**

### CODE REVIEW

File: string_cases/lib/string_cases.ex [hunk 1]
```diff
@@ -0,0 +1,41 @@
+defmodule StringCases do
+  @moduledoc """
+  A collection of functions for changing the casing of string values
+  """
+
+  @doc """
+  Given a string return a properly CamelCased string.
```
Comment (gemantzu) [inline string_cases/lib/string_cases.ex]: Add a ',' after the word string

File: string_cases/lib/string_cases.ex [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  # Examples
+
+  iex> StringCase.camel_case("Elixir-School is awesome!")
+  ElixirSchoolIsAwesome
+  """
+  def camel_case(input) do
+  end
+
+  @doc """
+  Snake_case the inputed string value.
```
Comment (gemantzu) [inline string_cases/lib/string_cases.ex]: Maybe make it `snake_case` to be more close to the actual result?

File: string_cases/lib/string_cases.ex [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  The resulting string should not include any punctuation other than underscores.
+  Spaces are replaced with underscore and words are downcased.
```
Comment (gemantzu) [inline string_cases/lib/string_cases.ex]: underscores

File: string_cases/lib/string_cases.ex [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  # Examples
+
+  iex> StringCase.snake_case("Elixir-School is awesome!")
+  elixir_school_is_awesome
+  """
+  def snake_case(input) do
+  end
+
+  @doc """
+  Everyone's favorite!  Wavy case alternates upper- and downcased letters started with a capitalize letter.
```
Comment (gemantzu) [inline string_cases/lib/string_cases.ex]: There is something wrong here with spaces and -, maybe try to find out what?

Comment (doomspork) [reply string_cases/lib/string_cases.ex]: @gemantzu I _think_ this is correct but I'm not an English major 😆 

I'm saying upper and lowercased  `upper-` being the prefix.  I can change this.

**Comment by doomspork** [comment]
@gemantzu feedback incorporated 👍

**Pull request merged**